### ISSUE TITLE
Alt+k and Alt+l work when dialog is open or not using keyboard nav

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#359]
**Test device**
Lovelace
**Operating System**
ChromeOS
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/513
The alt+k and alt+l commands work when the dialog for light spectrum is open, which seems pretty odd. It also works even if you haven't activated keyboard nav using tab.

**Visuals**
![lightspectrumaltl](https://user-images.githubusercontent.com/41024075/87208183-12560b80-c2cb-11ea-9b6d-ec995c6e368f.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_all_phet.html
Version: 1.5.0-rc.2 2020-07-09 20:53:24 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe -
> It also works even if you haven't activated keyboard nav using tab.

This part seems OK to me. But being able to use the global hotkeys while a Dialog is feels incorrect to me. This will probably also happen for the About Dialog, PhET Menu, Keyboard Shortcuts Dialog, and others.

@terracoda what do you think about this, should you be able to use "global" hotkeys while a modal window is open?

**Comment by terracoda** [comment]
@jessegreenberg, there should be no access (keyboard access or otherwise) to the content underneath a dialog when the dialog is open. A learner should have to exit the dialog to get back to the content with any keyboard commands.

There are techniques for locking out access to the content. I think we resolved that problem in previous work.

Are you able to "turn off" the hot keys when the dialog is open?

And I agree, that there is no need for "any keyboard activation" to have access to the global hotkeys. Learners should be able to have their keyboard focus on a radiobutton and still pause and step forward with the hotkeys.

When any dialog is open, however, there should be no access to content outside the open dialog.

**Comment by jessegreenberg** [comment]
OK, that makes sense to me.  Our previous work on this was just to remove items from the focus order when a modal dialog/menu is active. I am not sure yet how to prevent global hotkeys in this case but I will investigate.

**Comment by jessegreenberg** [comment]
The above commits provide a solution for this, and they are on master. @KatieWoe can you please confirm they are working as you would expect now?

**Comment by KatieWoe** [comment]
Look fixed on master

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, I will move those changes to the release branches then.

**Comment by KatieWoe** [comment]
This looks fixed in https://github.com/phetsims/QA/issues/520 on ChromeOS. Closing
